app/models/metric/processing.rb

```diff
@@ -28,6 +28,7 @@ module Metric::Processing
     ContainerProject,
     ContainerReplicator,
     ContainerService,
+    ContainerImage,
```

Comment:
It's obvious to me if we need this here

Reply:
`process_derived_columns` is used by metric rollups which is what calls `VALID_PROCESS_TARGETS` so I think he does need this